build.py

```diff
@@ -63,20 +64,21 @@ def copy(src: Path, dest, ignore=()):
 
 
 def install_requirements(py_executable):
-    with open("requirements.txt", "r") as file:
-        required_modules = file.readlines()
+    os.environ['PATH'] += os.pathsep + str(Path(site.getusersitepackages()).parent / "Scripts")
 
     installed_modules = []
-    for m in required_modules:
```

Comment:
revert with `required_modules` back
add `if not m: continue`
rename `m` -> `module`